src/index.ts

```diff
@@ -4,13 +4,22 @@ import _ from 'lodash'
 import DocumentReference = firebase.firestore.DocumentReference
 import DocumentSnapshot = firebase.firestore.DocumentSnapshot
 import QuerySnapshot = firebase.firestore.QuerySnapshot
+import Query = firebase.firestore.Query;
+
+const cacheTimeout = 3000;
```

Comment:
I think it would be better to expose this as a parameter (and even disable cache)